### PR TITLE
Add basic testing with Molecule and Travis-CI

### LINKED ISSUE #3: Testing Environment
Decide what’s the best method to run unit and integration tests and implement basic tests that are extended as the development continues.
 
Potential options:
* Molecule: https://github.com/metacloud/molecule
* Test Kitchen

Implement some basic testing with TravisCI to catch syntax, linting and gross errors.

### PR DESCRIPTION
**Pull request by bobapple**
closes #3

### CODE REVIEW

**Comment by bobapple** [comment]
This PR introduces Molecule as testing framework. It seems reasonable to me compared to other things I used with Puppet (Beaker) and Chef (TestKitchen). Currently it tests only some things on Centos7 and Ubuntu 16.04. Testing will be extended as necessary.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+this roadmap when you start contributing to the project.
+
+Before starting your work on this module, you should [fork the project] to your GitHub account. This allows you to
+freely experiment with your changes. When your changes are complete, submit a [pull request]. All pull requests will be
+reviewed and merged if they suit some general guidelines:
+
+* Changes are located in a topic branch
+* For new functionality, proper tests are written
+* Changes should not solve certain problems on special environments
+* Your change does not handle third party software for which dedicated Puppet modules exist
```
Comment (RossBarnie) [inline CONTRIBUTING.md]: "Puppet" should probably be "Ansible"

Comment (bobapple) [reply CONTRIBUTING.md]: You're absolutely right

**Comment by RossBarnie** [comment]
Ran this locally without issue.

In terms of platform support should this role not also support the most recently released LTS version of ubuntu (18.04)?

**Comment by bobapple** [comment]
Yes it will support Ubuntu 18.04 as well. The intention behind this PR is to create a starting point for testing in general. It certainly needs to be completed as the role grows.

**Review by aflatto — APPROVED** [review]

**Pull request merged**